client/components/theme-type-badge/tooltip.tsx

```diff
@@ -163,11 +178,20 @@ const ThemeTypeBadgeTooltip = ( {
 						),
 					}
 			  );
-	} else if ( type === WOOCOMMERCE_THEME ) {
-		message = isIncludedCurrentPlan
-			? translate( 'This WooCommerce theme is included in your plan.' )
-			: createInterpolateElement(
-					translate( 'This WooCommerce theme is included in the <Link>Business plan</Link>.' ),
+	} else if ( type === BUNDLED_THEME ) {
+		if ( bundleSettings[ themeSoftware ] ) {
+			const bundleName = bundleSettings[ themeSoftware ].name;
+
+			if ( isIncludedCurrentPlan ) {
+				message = translate( 'This %(bundleName)s theme is included in your plan.', {
+					args: { bundleName },
+				} );
+			} else {
+				message = createInterpolateElement(
+					translate( 'This %(bundleName)s theme is included in the <Link>Business plan</Link>.', {
+						args: { bundleName },
+						textOnly: true,
+					} ),
```

Comment:
We should add some translation comments to make it clearer what values we expect to pass in for `bundleName`.

Reply:
Sure! Added in https://github.com/Automattic/wp-calypso/pull/84483/commits/707dffc27cde6ca5852c39af4746d6466bd58a78